workflow-templates/npm-audit-fix.yml

```diff
@@ -7,15 +7,25 @@ name: npm audit fix and compile
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '5 2 * * 0'
```

Comment:
```suggestion
  schedule:
    # At 2:30 on Sundays
    - cron: '30 2 * * 0'
```

Reply:
(Only for clarity and because we have other jobs at 2:05 :grin: )

Reply:
done